core/pkg/ingress/annotations/authreq/main.go

```diff
@@ -129,9 +131,22 @@ func (a authReq) Parse(ing *extensions.Ingress) (interface{}, error) {
 
 	return &External{
 		URL:             str,
+		Host:            stripPort(ur.Host),
 		SigninURL:       signin,
 		Method:          m,
 		SendBody:        sb,
 		ResponseHeaders: h,
 	}, nil
 }
+
+// TODO: Remove after upgrade to Go 1.8
+func stripPort(hostport string) string {
```

Comment:
for someone reading the git history, it might be worth rewriting history in your PR so the "release 0.9-beta.4" commit is last. otherwise this seems fine.